tools/tpm2_policycphash.c

```diff
@@ -53,9 +53,9 @@ static bool on_option(char key, char *value) {
 static bool tpm2_tool_onstart(tpm2_options **opts) {
 
     static struct option topts[] = {
-        { "policy",  required_argument, NULL, 'L' },
-        { "session", required_argument, NULL, 'S' },
-        { "cphash",  required_argument, NULL,  0  },
```

Comment:
leave this and map cphash and cphash input to byte value 0 for the argument handler.

Reply:
agreed. fixed in latest push.